starwars/src/components/StarWars.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './StarWars.css';
+import StarWarsJSON from './StarWarsJSON.json';
+import Character from './Character.js'
+
+
+class StarWars extends React.Component {
+    
+    render() {
+        
+        return (
+            <ol>
+            {StarWarsJSON.map( character => {
```

Comment:
In this component, you should be getting the JSON through props as opposed to through the file you're importing